templates/repo/view_list.tmpl

```diff
@@ -28,6 +28,10 @@
 						{{end}}
 				</a>
 				<span class="grey has-emoji">{{RenderCommitMessage .LatestCommit.Summary .RepoLink $.Repository.ComposeMetas}}
+                {{if IsMultilineCommitMessage .LatestCommit.Message}}
+                <button class="basic compact mini ui icon button commit-button"><i class="ellipsis horizontal icon"></i></button>
+                <p class="commit-body" style="display: none;">{{RenderCommitBody .LatestCommit.Message $.RepoLink $.Repository.ComposeMetas}}</span>
```

Comment:
This should not be inside a paragraph, but instead inside a `<pre>` like in the other template (in order for newlines to be rendered properly).

Reply:
Yep, I forgot to change this one once I updated the other.